packages/block-library/src/site-logo/edit.js

```diff
@@ -517,11 +523,12 @@ export default function LogoEdit( {
 
 	const mediaReplaceFlowProps = {
 		mediaURL: logoUrl,
+		name: ! logoUrl ? __( 'Choose Site Logo' ) : __( 'Replace' ),
 		onSelect: onSelectLogo,
 		onError: onUploadError,
 		onRemoveLogo,
 	};
-	const controls = canUserEdit && logoUrl && ! temporaryURL && (
+	const controls = (
```

Comment:
```suggestion
	const controls = canUserEdit && (
```

I think it's necessary to keep canUserEdit, otherwise users who can't reset/update the site icon will see the toolbar menu unintentionally when they insert this block.

Reply:
Yes, we need to keep capability checks in place.

Reply:
good catch, thanks.